### ISSUE TITLE
Hotkeys interfere with keyboard input

### ISSUE THREAD

**Issue by hooby3dfx** [libretro/dosbox-libretro#52]
If you are using a keyboard to run commands on the dos prompt the user will inadvertently toggle various retroarch options (such as pause with the p key).

What are the best workarounds?

What would the actual solutions be?

**Comment by hooby3dfx** [comment]
There is a pretty good workaround for this, which is to set input_enable_hotkey_btn in retroarch.cfg.

Maybe something along those lines could be a core option?

**Comment by hizzlekizzle** [comment]
There's another thing you can use now called "game focus mode" or something like that. It works like mouse/kbd capture on e.g. Virtualbox. That is, it forwards all of the input events directly to the core instead of letting the frontend listen to them first.

**Comment by Gagert** [comment]
I opened the retroarch.cfg file in a text editor, scrolled past the input_player1 section and typed in "nul" for each and every keyboard commands besides F1 (RetroArch menu) and volume up and down.

Now I can type in what I want without accidentally hitting a keyboard hotkey is dosbox libretro.

**Comment by hooby3dfx** [comment]
There are a lot of great solutions to this problem that can be used.
I think this issue can be closed once the workarounds are documented.

Or should one of them be there by default for this core?

**Comment by vol-2** [comment]
> There are a lot of great solutions to this problem that can be used. I think this issue can be closed once the workarounds are documented. Or should one of them be there by default for this core?

Yes, I think so. I just had this issue. I found this thread, but it seems like a pain to go through and remove every single key. The other option should be a core option for sure.

**Comment by hizzlekizzle** [comment]
game focus mode is the solution.

I do think it would be nice if there were a way for a core to communicate to the frontend that it's going to need the keyboard, but in the meantime having a hotkey to redirect all of the keyboard events directly to the core (i.e., keyboard capture, i.e., game focus mode) seems adequate.

**Comment by vol-2** [comment]
Thanks, I found the focus mode. :)

Yeah, that's what I meant. It would be nice if during dosbox all keys were available except some key combo to bring up the menu. Alt-F1 or something unlikely to be used.

**Comment by jasiup** [comment]
but how to disable "Winkey" in "game focus mode on"
